### PR TITLE
53_HeaderRendersCorrectView

### PR DESCRIPTION
**Pull request by JoMuelle**
-Changed the Header, now a Function that adjusts depending on the view
-Header now contains Login or Register button depending on view
-Deleted Login/Register button from Register/Login View

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
  * Happy coding!
  * React Template by Lucas Pelloni
  * Overhauled by Kyrill Hux
  */
+
+
 const App = () => {
   return (
     <div>
-      <Header height="100"/>
```
Comment (daveking7) [inline src/App.js:12]: could we just include height=100 in the header?

File: src/components/views/Game.js [hunk 2]
```diff
@@ -96,13 +97,16 @@ const Game = () => {
   }
 
   return (
+      <React.Fragment>
+          <Header heigh="100"/>
```
Comment (daveking7) [inline src/components/views/Game.js:101]: height

File: src/components/views/Game.js [hunk 3]
```diff
@@ -96,13 +97,16 @@ const Game = () => {
   }
 
   return (
+      <React.Fragment>
```
Comment (daveking7) [inline src/components/views/Game.js:100]: What does React.Fragment do?

File: src/components/views/Header.js [hunk 4]
```diff
@@ -11,18 +14,68 @@ import "styles/views/Header.scss";
...
+                    <Button
+                            onClick={() => goToRegistration()}
+                    >
+                        Register Account
+                    </Button>
+                </div>
+            </div>
+        }
+        if(props.view=="register"){
+            return     <div className="header container" style={{height: props.height}}>
```
Comment (daveking7) [inline src/components/views/Header.js:54]: Why don't we just write height: 100 here? It should not change throughout the entire website?

**Review by daveking7 — CHANGES_REQUESTED** [review]
What does React.Fragment do? Define the height of the header once and for all -> only one possibility for a bug.

File: src/components/views/Game.js (hunk 3 above)
Comment (JoMuelle) [reply src/components/views/Game.js:100]: Allows for passing more than 1 html object

**Pull request closed without merging**